pages/common/streamlit.md

```diff
@@ -0,0 +1,20 @@
+# streamlit
+
+> Framework for creating interactive, data-driven web apps in Python.
+> More information: <https://docs.streamlit.io/>.
+
+- Check Streamlit installation:
+
+`streamlit hello`
+
+- Run your Streamlit application:
+
+`streamlit run {{project_name}}`
+
+- See all supported Streamlit commands:
```

Comment:
```suggestion
- Check for Streamlit installation:

`streamlit hello`

- Run your Streamlit application:

`streamlit run {{project_name}}`

- Display help:
```
For help and version commands we generally use this generic wording across our pages (for easy translations).

Reply:
Understood, rest of document seems in a generic format. so I'll make the given change only